core/src/main/scala/org/apache/spark/ui/UIUtils.scala

```diff
@@ -265,6 +266,7 @@ private[spark] object UIUtils extends Logging {
       <head>
         {commonHeaderNodes}
         {if (useDataTables) dataTablesHeaderNodes else Seq.empty}
+        <link rel="shortcut icon" href="/static/spark-logo-77x50px-hd.png"></link>
```

Comment:
ditto

Reply:
@guoxiaolongzte don't you need to change this too?